packages/flutter/lib/src/widgets/linked_text.dart

```diff
@@ -0,0 +1,776 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math' as math;
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/gestures.dart';
+
+import 'basic.dart';
+import 'framework.dart';
+import 'text.dart';
+
+/// A callback that passes a [String] representing a link that has been tapped.
```

Comment:
nit from style guide: `Typedef dartdocs should usually start with the phrase "Signature for…​".`